src/js/core/methods/helpers/stellarSignTx.js

```diff
@@ -87,10 +87,10 @@ const transformOperation = (op: StellarOperation): ?StellarOperationMessage => {
                 amount: op.amount,
             };
 
-        case 'pathPayment':
+        case 'pathPaymentStrictReceive':
             validateParams(op, [{ name: 'destAmount', type: 'amount', obligatory: true }]);
```

Comment:
I am curious why only `destAmount` is checked here, but `sendMax` is not?

Reply:
probably somebody forgot to add this? I guess this is not a big deal, firware will run its own validations.